nucypher/cli/painting.py

```diff
@@ -898,8 +905,8 @@ def paint_bidder_status(emitter, bidder):
     message = f"""
 WorkLock Participant {bidder.checksum_address}
 =====================================================
-Total Bid ............ {bidder.get_deposited_eth}
-Available Refund ..... {bidder.available_refund}
+Total Bid ............ {prettify_eth_amount(bidder.get_deposited_eth)}
+Available Refund ..... {prettify_eth_amount(bidder.available_refund)}
```

Comment:
What about including available compensation also?

Reply:
Not sure about this because it will be for very few addresses and may confuse other bidders

Reply:
Will there ever be a practical need to obtain this information as *any* kind of user?

Reply:
Added printing of compensation if there is any